### ISSUE TITLE
epubcheck doesn't recognize reserved a11y namespace

### ISSUE THREAD

**Issue by acabal** [IDPF/epubcheck#829]
epubcheck version v4.0.2
Ubuntu 16.04

We're trying to implement the [epub a11y spec](https://idpf.github.io/epub-vocabs/package/a11y/) in our ebooks, where in [section 1.2](https://idpf.github.io/epub-vocabs/package/a11y/#sec-ref) it states that:

>  The prefix "a11y:" is reserved for use with properties in this vocabulary and does not have to be declared in the Package Document.

However if you run `epubcheck` on an epub which uses the `a11y` prefix, it outputs this error:

> ERROR(OPF-028): /tmp/my-ebook.epub/epub/content.opf(22,59): Undeclared prefix: 'a11y'.

This can be worked around by declaring the `a11y` namespace in the package OPF, but according to the spec we shouldn't have to.

It's unclear to me if the a11y spec depends on epub 3.1 (which I know is not supported by `epubcheck` at the moment) but in case it isn't, I thought I'd open this issue.

**Comment by acabal** [comment]
D'oh, this is #810. Closed as a duplicate, sorry!
